### PR TITLE
Nitty nit nit.

### PR DESCRIPTION
**Pull request by gburtini**
1. Self-referential linting, instead of partial lint w/ Airbnb rules.
2. There was no `yarn.lock`.
3. Prettier reformatted one of the bigger objects.

I promoted the patch version. I initially wasn't going to because this is all dev-time stuff, but Google convinced me that `yarn.lock` should be published the `npm` and that could plausibly change the dependency graph.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "airbnb"
+  "extends": "./index.js"
```
Comment (gburtini) [inline .eslintrc:2]: This has been tested. It works as expected. You'll see an example of it under [Using a Configuration File](http://eslint.org/docs/user-guide/configuring) in the yarn docs.

**Review by jakeorr — APPROVED** [review]
LGTM, although I am curious what prompted these changes in the first place.

**Comment by gburtini** [comment]
@jakeorr: I read a blog post that criticized people for having incorrect `.npmignore` (you should ignore all non-{built, required, end use pieces}) and went and checked all my projects. This one did not have the problem, but while I was being nitpicky I decided to clean it up.

**Comment by jakeorr** [comment]
cool cool, I can appreciate some good nit picking

**Pull request merged**